src/main/kotlin/com/sourcegraph/cody/config/notification/CodySettingChangeListener.kt

```diff
@@ -20,7 +20,7 @@ class CodySettingChangeListener(project: Project) : ChangeListener(project) {
         object : CodySettingChangeActionNotifier {
           override fun afterAction(context: CodySettingChangeContext) {
             // Notify JCEF about the config changes
-            javaToJSBridge?.callJS("pluginSettingsChanged", ConfigUtil.getConfigAsJson())
+            javaToJSBridge?.callJS("pluginSettingsChanged", ConfigUtil.getConfigAsJson(project))
```

Comment:
Nothing listens to this, so can we simply remove it?

Reply:
For code listening on that you would need to look at `sourcegraph/sourcegraph`

Reply:
As long as it is used, that's fine. Thanks for the pointer.